index.html

```diff
@@ -0,0 +1,236 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+            rel="stylesheet"
+            integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+            crossorigin="anonymous">
+        <link rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+        <link rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Lato">
+        <link rel="stylesheet" type="text/css" href="styles.css" />
+        <title>Dev Courses</title>
+    </head>
+    <body>
+        <header>
+            <img id="hero-area-bg-img" src="images/hero-bg.jpg" alt="">
+            <div id="header-nav-plus-content">
+                <nav>
+                    <ul id="upper-nav">
+                        <li id="hamburger"><a><i class="bi bi-list"></i></a></li>
+                        <li class="nav"><a href="#"><i class="bi bi-twitter"></i></a></li>
+                        <li class="nav"><a href="#"><i class="bi bi-facebook"></i></a></li>
+                        <li class="nav"><a class="upper-nav-anchor" href="#"><span>My
+                                    Page</span></a></li>
+                        <li class="nav"><a href="#"><span>English</span></a></li>
+                        <li class="nav"><a href="#"><span>Login</span></a></li>
+                    </ul>
+                </nav>
+                <nav>
+                    <ul id="lower-nav" class="nav">
+                        <li><img src="images/logo.png" alt="Dev Courses Logo"></li>
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="#">Older</a></li>
+                        <li><a href="#">Trending</a></li>
+                        <li><a href="#">Your Couses</a></li>
+                    </ul>
```

Comment:
- [ ] Once you click on About  menu twice, the link seem to be broken, kindly fix that issue.

Reply:
- [ ] As pointed out by the previous reviewer, when the About link is clicked twice, it gives an error. Please kindly check on that and fix it.

Reply:
Hi. Do you mean the mobile dropdown? or the About link in the lower nav section ( in the about page or in the main page? ) ? because I am checking on my side and there is no problem

Reply:
- [ ] Hello. Now when I click on it once it gives me '404 There isn't a GitHub Pages site here.". Please kindly work on that.